include/fmt/core.h

```diff
@@ -434,30 +434,30 @@ template <typename Char> class basic_string_view {
   }
 
   // Lexicographically compare this string reference to other.
-  int compare(basic_string_view other) const {
+  FMT_CONSTEXPR int compare(basic_string_view other) const {
```

Comment:
This function uses `std::char_traits::compare()` which is [available at compile time starting from C++17](https://en.cppreference.com/w/cpp/string/char_traits/compare), while `FMT_CONSTEXPR` is replaced with `constexpr` starting from C++14 (with some exceptions). So for C++14, this code is invalid at compile-time, but no one uses it with this standard anyway, so there is no compilation error.

Reply:
Thanks for clarification.

Reply:
Let's do this then
https://github.com/llvm-mirror/libcxx/blob/master/include/__string#L114